blueprints/10-l4tm_packages/blueprint.py

```diff
@@ -135,6 +156,31 @@ def _load_data():
             _data.update(dict((pkg['Package'], pkg) for pkg in tmp))
 
 
+def get_all_mirrors():
+    """
+        Return a list of sources.list entries from the tmms config parameters.
+    """
+    mirror = BP.config['DEBIAN_MIRROR']
+    release = BP.config['DEBIAN_RELEASE']
+    areas = list(BP.config['DEBIAN_AREAS'])
+    areas = ' '.join(areas)
+
+    main_source = 'deb {url} {release} {areas}'\
```

Comment:
split the lines on an open left parenthesis, don't use the line continuation:

main_source = 'deb {url} {release} {areas}'.format(

Reply:
Ah... I hate this. But okay.

Reply:
Yes, but pylint hates the continuation character :-)